app/gateway/changelog.md

```diff
@@ -3,6 +3,43 @@ title: Kong Gateway Changelog
 no_version: true
 ---
 
+<!-- vale off -->
+
+## 2.8.1.2
+**Release Date** 2022/06/28
+
+### Fixes
+
+#### Enterprise
+
+* Fixed an issue in hybrid mode where, if a service was set to `enabled: false` and that service had a route with an enabled plugin, any new data planes would receive empty configuration.
+* Fixed a timer leak that occurred when `worker_consistency` was set to `eventual` in `kong.conf`.
+The issue caused timers to be exhausted and failed to start any other timers used by Kong Gateway, showing the error `too many pending timers`.
+
+#### Kong Manager and Dev Portal
+
+* Fixed an issue where Kong Manager did not display all Dev Portal developers in the organization.
+* Fixed an issue with displaying developer role assignment in Kong Manager.
```

Comment:
Does this one mean that it just didn't display what roles a developer was assigned to? If so, maybe something like this would work? I think the "displaying" part is throwing me off.
```suggestion
* Fixed an issue that prevented developer role assignments from displaying in Kong Manager.
```